### ISSUE TITLE
I broke Load More

### ISSUE THREAD

**Issue by jdcauley** [isratmir/Instagram_PHP_API#2]
Load more is broken, tried starting from scratch to no success

https://github.com/jdcauley/asthmagram

**Comment by isratmir** [comment]
In ajax.php : 
   change this :   $instagram = new Instagram('client_id');
   with this:         $instagram = new Instagram('099ddb4d09e648a6b11448961a99256b');

**Comment by jdcauley** [comment]
Still doesn't work

## 

Jordan Cauley
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by jdcauley** [comment]
Showing my ignorance.  I have your load more working now (thank you very much).

I have no idea how to include my foreach and if statements in the ajax append

**Comment by isratmir** [comment]
You're welcome Jordan!
My English isn't very good but i try to explain you. There is three main files: index.php, ajax.php, app.js. 
From index.php through app.js we send request to ajax.php with parameters. If request is successfull ajax.php returns response with array to app.js. app.js forms HTML. this part of code: 
`$.each(data.dt, function(i) {
              $('.row').append('<div class="span4 box"><div class="padding"><img src="' + data.dt[i].images.standard_resolution.url + '"></div></div>');
 });`

**Comment by jdcauley** [comment]
If you go to http://jordancauley.com/asthmagram you can see were i'm running into the issue I need my ajax load to match my php load.

ie displaying likes, comments and the posting date.

I use if and foreach statements in the php but have no idea how to do that in the append js.

I pushed the updated code to the repo.

**Comment by isratmir** [comment]
If i got your point your problem is printing formatted date on dinamically uploaded content. Replace code in app.js with this:

```
$(document).ready(function() {
      $('#more').click(function() {
        var tag   = $(this).data('tag');
        var maxid = $(this).data('maxid');
        var months = [ "January", "February", "March", "April", "May", "June", "July", "August", "September","October", "November", "December" ];

        $.ajax({
          type: 'GET',
          url: 'ajax.php',
          data: {
            tag: tag,
            max_id: maxid
          },
          dataType: 'json',
          cache: false,
          success: function(data) {
            // Output data
            $.each(data.dt, function(i) {
              var date = new Date(parseInt(data.dt[i].created_time) * 1000);
              var postedDate = months[date.getMonth()]+' '+date.getDate()+', '+date.getFullYear()+' '+date.getHours()+':'+date.getMinutes();
              $('.photos').append('<div class="span4 box"><div class="padding"><a href="#'+ data.dt[i].id +'" role="button" data-toggle="modal"><img src="' + data.dt[i].images.standard_resolution.url + '"></a><div class="meta-profile"><a href="http://instagram.com/' + data.dt[i].user.username +'"><img alt="' + data.dt[i].user.username +'" src="' + data.dt[i].user.profile_picture +'"></a><span class="instameta pull-right">' +postedDate+ ' ' + data.dt[i].likes.count + ' <i class="icon-heart"></i> ' + data.dt[i].comments.count + ' <i class="icon-comment"></i></span></div></div></div> <div id="'+ data.dt[i].id +'" class="modal hide fade bigModal" tabindex="-1" role="dialog"><div class="modal-body"><div class="modal-img"><a href="#" target="_blank"><img class="modal-main-img" alt="' + data.dt[i].user.username + ' BeatAsthma" src="' + data.dt[i].images.standard_resolution.url + '"></a></div><div class="modal-profile"><button type="button" class="pull-right" data-dismiss="modal">x</button><a href="http://instagram.com/' + data.dt[i].user.username +'"><img class="profile-image" alt="' + data.dt[i].user.username +'" src="' + data.dt[i].user.profile_picture +'"></a><ul><li class="profile-username">' + data.dt[i].user.username +'</li><li class="profile-created-time">' + data.dt[i].created_time + '</li></ul><p class="caption">' + data.dt[i].caption.text +'</p><div class="likes"><p class="like-header"><i class="icon-heart"></i> ' + data.dt[i].likes.count + ' Likes</p></div></div></div>'); 
            });

            // Store new maxid
            $('#more').data('maxid', data.next_id);
          }
        });
      });
    });
```

**Comment by jdcauley** [comment]
That was one of my issues the other is displaying like sans comments.  If you click an image it displays a light box with comments etc

**Comment by jdcauley** [comment]
Also do you have a Paypal amazon wish list or something where I can send a little gift
